src/controller/python/chip/clusters/attribute.cpp

```diff
@@ -426,12 +440,15 @@ chip::ChipError::StorageType pychip_ReadClient_ReadEvents(void * appContext, Dev
         {
             params.mMinIntervalFloorSeconds   = pyParams.minInterval;
             params.mMaxIntervalCeilingSeconds = pyParams.maxInterval;
+            err                               = readClient->SendAutoResubscribeRequest(std::move(params));
+        }
+        else
+        {
+            err = readClient->SendRequest(params);
         }
-
-        err = readClient->SendRequest(params);
-        SuccessOrExit(err);
     }
 
+    readPaths.release();
```

Comment:
Again, leaks in the non-subscription case?

Reply:
Fixed, thanks